ac-dcd.el

```diff
@@ -583,18 +588,20 @@ dcd-client outputs candidates that begin with \"this\" when completing struct co
   "Goto the point where `ac-dcd-goto-definition' was last called."
   (interactive)
   (if (ring-empty-p ac-dcd-goto-definition-marker-ring)
-      (error "Marker ring is empty. Can't pop.")
+      (progn (message "Marker ring is empty. Can't pop.")
+             (ding))
     (let ((marker (ring-remove ac-dcd-goto-definition-marker-ring 0)))
       (switch-to-buffer (or (marker-buffer marker)
-                            (error "Buffer has been deleted")))
+                            (progn (message "Buffer has been deleted")
```

Comment:
2nd time this exact sequence is used. Better to define a function. What's wrong with the `error` function, anyway?